.github/workflows/cmake.yml

```diff
@@ -14,8 +14,10 @@ jobs:
         include:
           - os: ubuntu-20.04
             cxx: /usr/bin/g++-9
+            libint2-python: ON
           - os: macos-latest
             cxx: clang++
+            libint2-python: OFF
```

Comment:
@asadchev why not test python on macOS?

Reply:
i thought to have some workflow that test with python off.  you want treat all?